Exercise_B/index.html

```diff
@@ -30,5 +85,12 @@
     <!-- Delivery Div -->
 
     <!--Scripts required for bootstrap CDN-->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We" crossorigin="anonymous">
+
+<script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.3/dist/umd/popper.min.js" integrity="sha384-eMNCOe7tC1doHpGoWe/6oMVemdAVTMs2xqW4mwXrXsW0L84Iytr2wi5v2QjrP/xp" crossorigin="anonymous"></script>
+  
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/js/bootstrap.min.js" integrity="sha384-cn7l7gDp0eyniUwwAZgrzD06kc/tftFf19TOAs2zVinnD/C7E91j9yyk5//jjpt/" crossorigin="anonymous"></script>
+
+
   </body>
 </html>

```

Comment:
For the images, in your div's you have added a container. There shouldn't be a container class inside the classes for your image. As this is used to add items inside it. 
Image : <img src="./assets/images/flower-icon.png" also when adding in image and text you would need to add in the flexbox class and also flex row. I would suggest adding the image and then text in one block and then using fle-row